api/HTMLInputElement.json

```
@@ -1286,6 +1286,63 @@
           }
         }
       },
+      "setRangeText": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/setRangeText",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "27"
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "qq_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "7"
```

```suggestion
              "version_added": "6.1"
```
Per https://trac.webkit.org/changeset/131969/webkit and https://trac.webkit.org/browser/webkit/branches/safari-537.43-branch/Source/WebCore/html/HTMLInputElement.idl#L87, this was in WebKit 537.43, and so Safari 6.1.